src/Sylius/Bundle/CoreBundle/Processor/DummyProductCatalogPromotionProcessor.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Processor;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Sylius\Bundle\CoreBundle\Applicator\ProductCatalogPromotionApplicatorInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\Model\TaxonInterface;
+use Sylius\Component\Core\Repository\ProductRepositoryInterface;
+use Sylius\Component\Promotion\Model\CatalogPromotionInterface;
+use Sylius\Component\Taxonomy\Repository\TaxonRepositoryInterface;
+
+final class DummyProductCatalogPromotionProcessor implements ProductCatalogPromotionProcessorInterface
```

Comment:
Why do you name these things 'ProductCatalogPromotion' instead of just 'CatalogPromotion'?